pkg/util/workerqueue/workerqueue.go

```diff
@@ -77,7 +80,7 @@ func (wq *WorkerQueue) Enqueue(obj interface{}) {
 		runtime.HandleError(wq.logger.WithField("obj", obj), err)
 		return
 	}
-	wq.logger.WithField("key", key).Info("Enqueuing key")
+	wq.logger.WithField(wq.keyName, key).Info("Enqueuing")
```

Comment:
Should the workqueue have the keyname all the way through all it's logs?
(in which case, rather than store it, attach it as an extra `WithField` on `wq.logger` ?

Reply:
except the key is available to this method only, right?

Reply:
OIC! This is the wq.keyName - got it!